llvm/test/CodeGen/AArch64/apply-disjoint-flag-in-dagcombine.ll

```diff
@@ -0,0 +1,15 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=aarch64-none-eabi -verify-machineinstrs %s  -start-before=aarch64-isel -o - | FileCheck %s
```

Comment:
Why does this need `-start-before=aarch64-isel`? (Or `-verify-machineinstrs` for that matter)

Reply:
Cleared. This is not really needed.